### PR TITLE
Fixes #6 Created XML Layout for User and Admin Dashboard

### LINKED ISSUE #6: XML Layout for User Dashboard and Admin Dashboard
Create the XML layout for User Dashboard and Admin Dashboard based on the design provided in the [Figma](https://www.figma.com/design/jYm7XGidk1L3HNf9e4Otwn/myLeave?node-id=6-18&t=HjrOtyAfoLbxQDPn-1). Since the layout for both dashboards is the same, you can reuse the code from the User Dashboard layout for the Admin Dashboard.

Your layout must follow these guidelines:

1. Use Constraint Layout to ensure the layout is responsive to all screen sizes.
2. No need to create Recycler View layout and adapter, just add an empty Recycler View.
4. Adhere to the given Figma design
5. Follow the specified naming convention for each view group ID to maintain consistency and clarity.

### PR DESCRIPTION
**Pull request by ayushsaksena30**
## Overview

1. This PR fixes or fixes part of #6.
2. This PR does the following: Created the XML Layout for Admin and User Dashboard by using NestedScrollView and used constraints for responsiveness. Added Floating App button and necessary texts while adhering to the given Figma design.

## Essential Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [x] The PR is made from a branch that's **not** called "main/master".

## Proof that changes are correct
Screenshot 2024-06-08 at 9 49 12 AM

## PR Pointers

- If you need a review or an answer to a question, and don't have permissions to assign people, **leave a comment** like the following: "{{Question/comment}} @{{reviewer_username}} PTAL".
- Never force push. If you do, your PR will be closed.

### CODE REVIEW

File: app/src/main/res/layout/activity_adm_dashboard.xml [hunk 1]
```diff
@@ -1,10 +1,119 @@
...
+                android:fontFamily="@font/epilogue_bold"
+                android:text="@string/dashboard"
+                android:textColor="#171A1F"
+                android:textSize="24sp"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.45"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintVertical_bias="0.02">
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_adm_dashboard.xml:40]: Give proper ID to all views according to the convention, take reference from [here](https://stackoverflow.com/questions/12870537/android-naming-convention)

File: app/src/main/res/layout/activity_adm_dashboard.xml [hunk 2]
```diff
@@ -1,10 +1,119 @@
...
+                android:id="@+id/Btn_profile"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:background="null"
+                android:src="@drawable/ic_profile"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.90"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="@id/dashboard_title"
+                app:layout_constraintVertical_bias="0.05">
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_adm_dashboard.xml:54]: Do not use constraintVertical bias and constraintHorizontal bias , instead use appropriate margins to position them correctly. Correct it for all the view groups.

File: app/src/main/res/layout/activity_adm_dashboard.xml [hunk 3]
```diff
@@ -1,10 +1,119 @@
...
+
+            </ImageButton>
+
+            <TextView
+                android:id="@+id/pending_applications"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/pending_applications"
+                android:textColor="#171A1F"
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_adm_dashboard.xml:63]: Add the colour in colors.xml file and then use it.

**Review by Agrim2112 — CHANGES_REQUESTED** [review]
Make the changes as mentioned.

**Comment by ayushsaksena30** [comment]
@Agrim2112 I have made the necessary changes and pushed the new commit into the PR.

File: app/src/main/res/values/colors.xml [hunk 4]
```diff
@@ -6,4 +6,6 @@
     <color name="Title_color">#171A1F</color>
     <color name="subhead_color">#171A1F</color>
     <color name="btn_add_color">#379AE6</color>
+    <color name="black_title">#171A1F</color>
+    <color name="black_subhead">#171A1F</color>
```
Comment (Agrim2112) [inline app/src/main/res/values/colors.xml:10]: Mention the colour only once and then use it, remove the redundant ones.

**Comment by ayushsaksena30** [comment]
@Agrim2112 I have removed the duplicate labels and made the changes.

**Pull request merged**